packages/repository/src/common.ts

```diff
@@ -0,0 +1,59 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: @loopback/repository
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+/**
+ * Common types/interfaces such as Class/Constructor/Options/Callback
```

Comment:
I have proposed to rename this file to `src/common-types.ts` to make the intent more clear. I am not sure if you are intentionally keeping the old name `src/common.ts` or not. I am ok with either.